tests/Entities/StageEntityTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\Entities;
+
+use Portal\Entities\StageEntity;
+use \Exception;
+use PHPUnit\Framework\TestCase;
+use TypeError;
+
+class StageEntityTest extends TestCase
+{
+    public function testNewStageEntitySuccess()
```

Comment:
docBlocks for each test

Reply:
There aren't any in other tests, however, I think in this case they should be, so have added them.